scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -325,24 +325,20 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                     return;
                 }
 
-                function fieldErr(err) {
-                    return $interpolate(err)({field: fieldName});
-                }
-
                 if (errors.date) {
-                    return fieldErr(gettext('{{ field }} date is required!'));
+                    return gettext('{{ field }} date is required!', {field: fieldName});
```

Comment:
Remove empty spaces between translation variables in the code.

Reply:
What's wrong here?
The text extracted looks OK:
#: scripts/apps/authoring/authoring/directives/AuthoringDirective.js:329
msgid "{{ field }} date is required!"
msgstr ""

Reply:
> `'{{ field }} date is required!'`

There are spaces between the word "field"  and curly brackets in `{{ field }}`. They are not needed and might confuse the translators, so I suggest to remove it.

Reply:
Do you mean:
return gettext('{{field}} date is required!', {field: fieldName});
?

Reply:
Yes :)